### ISSUE TITLE
Confirmation regarding the CSVs since no ReadMe.md found for them

### ISSUE THREAD

**Issue by moha23** [ahmed1996said/gardnet#3]
Hi,

Thank you for making your work public. I wanted to confirm few things regarding the files you have shared as I couldn't find it clarified explicitly. 

In your codes you refer to a file 'train_256_bbs_final.csv' which I couldn't find, but there are train_i and val_i csv files for i = 0 to 4. Am I right in assuming that these are 5 different folds of the full dataset (plus the test.csv)? So we can use anyone of these train_i+val_i+test csv's instead of  'train_256_bbs_final.csv'? These csv's contain the bounding boxes generated using the OD segmentation and crop code.

Hope I got this right, thanks!

**Comment by ahmed1996said** [comment]
Hi @moha23,

Thanks for your questions, and apologies for any confusion.

It seems like the file `train_256_bbs_final.csv` refers to the original dataset that we used before splitting them into 5 folds. I'm seeing that this file was generated from this [notebook](https://github.com/ahmed1996said/gardnet/blob/main/notebooks/Optic%20Disc%20Segmentation%20and%20Crop.ipynb).

For training, yes, your assumption is correct. `train_i.csv` and `val_i.csv` refer to the ith fold for training and validation, respectively.
In short, the `train_256_bbs_final.csv` contains the entire dataset before being split into folds and training/validation/test that we have provided on Google Drive.

Hope this helps.
